### ISSUE TITLE
About pretrained model

### ISSUE THREAD

**Issue by huangqianfirst** [avalonstrel/GatedConvolution_pytorch#8]
Could you please update the complete pretrained model. I've test the corase model.The result is not good.
![44](https://user-images.githubusercontent.com/11473038/49500972-d10aa800-f8ac-11e8-954d-da80a3ac81df.png)
![39](https://user-images.githubusercontent.com/11473038/49500975-d1a33e80-f8ac-11e8-885f-abf258e298a5.png)
Thank you very much.

**Comment by UPML** [comment]
+1 
I have the same results with pretrained model. 
And also what about dark areas at generated images? 
![40](https://user-images.githubusercontent.com/7060488/49511845-55552f00-f89d-11e8-8267-968b9d4c2cac.png)
![44](https://user-images.githubusercontent.com/7060488/49511853-5a19e300-f89d-11e8-8f6e-81e4d335e0e1.png)
![39](https://user-images.githubusercontent.com/7060488/49512238-5b97db00-f89e-11e8-9d3e-213675671c02.png)

**Comment by avalonstrel** [comment]
Thanks, the model is trained before the model structure changed, so I will retrain a model using the current script.

**Comment by UPML** [comment]
Ok, thanks. How much GPU time do you spent on training?

**Comment by an1018** [comment]
@avalonstrel I use the TensorFlow version and alsoencountered this problem.Have you solve this problem,and what  do you think causes the result?

**Comment by zhangbaijin** [comment]
why i download the pretrained models, but it is broken?? and i can not open it ?

**Comment by huangqianfirst** [comment]
I have trained 'Free_from Image Inpainting with Gated Convolution' from scratch  ,here is my result
https://github.com/huangqianfirst/Free_from-Image-Inpainting-with-Gated-Convolution

**Comment by sporterman** [comment]
@huangqianfirst how did you create mask file while training?

**Comment by janzd** [comment]
@huangqianfirst Did you use the implementation in this repository? If yes, what settings did you use to be able to produce such good results as you show?

**Comment by huangqianfirst** [comment]
@kurapan .No ,i  implemente by myself based DeepFillv1 (https://github.com/JiahuiYu/generative_inpainting).

**Comment by huangqianfirst** [comment]
@sporterman .You can get the advice about how to create mask from here:https://github.com/JiahuiYu/generative_inpainting/issues/62.

**Comment by DongHwanJang** [comment]
@huangqianfirst Is that a valid set result? and also do have any idea to open the implementation?

**Comment by huangqianfirst** [comment]
@DongHwanJang . 1. It's test results. 2. Not yet.

**Comment by DongHwanJang** [comment]
@huangqianfirst how long did it take to train?

**Comment by huangqianfirst** [comment]
@DongHwanJang . About five days using Titan X, one GPU.
